Extension/src/LanguageServer/ui.ts

```diff
@@ -36,10 +36,18 @@ interface ConfigurationStatus {
 }
 
 export class UI {
-    private configStatusBarItem: vscode.StatusBarItem;
-    private browseEngineStatusBarItem: vscode.StatusBarItem;
-    private intelliSenseStatusBarItem: vscode.StatusBarItem;
-    private referencesStatusBarItem: vscode.StatusBarItem;
+    private configStatusBarItem!: vscode.LanguageStatusItem;
+    private browseEngineStatusBarItem!: vscode.LanguageStatusItem | undefined;
+    private intelliSenseStatusBarItem!: vscode.LanguageStatusItem | undefined;
+    private referencesStatusBarItem!: vscode.LanguageStatusItem | undefined;
+    // This is a duplicate of what's in client.ts
+    // TODO: Confirm whether the orignal can be reused here
+    private documentSelector: vscode.DocumentFilter[] = [
```

Comment:
Yeah, I see this used in 2 places in client.ts -- I'm thinking it could go in some common/util file as an exported variable. We want to try to reduce the size of client.ts and it doesn't seem directly related to the client so it probably shouldn't go there.